src/actions/channels.js

```diff
@@ -467,6 +467,35 @@ export function getChannelAndMyMember(channelId) {
     };
 }
 
+export function getChannelTimezones(channelId) {
+    return async (dispatch, getState) => {
+        dispatch({type: ChannelTypes.GET_CHANNELS_TIMEZONE_REQUEST}, getState);
```

Comment:
`dispatch` doesn't actually take a second argument, so we don't need to pass `getState` into it

Reply:
done